apps/code-examples/src/app/features/box.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { BoxDemoComponent as BasicComponent } from '../code-examples/layout/box/basic/box-demo.component';
+import { BoxDemoModule as BasicModule } from '../code-examples/layout/box/basic/box-demo.module';
+
+const routes: Routes = [
+  {
+    path: 'basic',
+    component: BasicComponent,
+  },
+];
+
+@NgModule({
+  imports: [RouterModule.forChild(routes)],
+  exports: [RouterModule],
+})
+export class BoxRoutingModule {}
+
+@NgModule({
+  imports: [BasicModule, BoxRoutingModule],
+})
+export class BoxModule {}
```

Comment:
I figured we could create modules for each library (not necessarily each component). So this would be the `layout.module.ts`

(The `action-bars.module.ts` was probably a bad component to start with since its library and component are named the same thing.)

Reply:
We'd follow this same structure with the playground and integration SPAs as well.